build.py

```diff
@@ -131,7 +131,7 @@ def _paths(*parts: str) -> list[str]:
 
     orig = dict(os.environ)
     os.environ.update(
-        CPP_FLAGS=" ".join(f"-I{path}" for path in _paths("lib")),
+        CFLAGS=" ".join(f"-I{path}" for path in _paths("include")),
```

Comment:
lol this was completely broken, no idea how it worked before

Reply:
Sorry, missed that before, but lol.